spec/index.html

```diff
@@ -1449,30 +1450,36 @@ <h3>Algorithm</h3>
         <a>string</a> <var>position</var> as inputs.</p>
 
       <ol>
-        <li id="hrbn-1">Set the <var>identifier</var> to use for <var>related</var>, preferring
-          first the canonical identifier for <var>related</var> if issued, second
-          the identifier issued by <var>issuer</var> if issued, and last, if
-          necessary, the result of the
-          <a href="#hash-1d-quads">Hash First Degree Quads algorithm</a>,
-          passing <var>related</var>.
+        <li id="hrbn-1">Initialize a <a>string</a> <var>input</var> to the value of
+          <var>position</var>.</li>
+        <li id="hrbn-2">If <var>position</var> is not <code>g</code>, append
+          <code>&lt;</code>, the value of the <a>predicate</a> in
+          <var>quad</var>, and <code>&gt;</code> to <var>input</var>.</li>
+        <li id="hrbn-3">If there is a canonical identifier for <var>related</var>,
+          or an identifier issued by <var>issuer</var>,
+          append the string `_:`, followed by the that identifier to <var>input</var>.
```

Comment:
For clarity:

```suggestion
          append the string `_:`, followed by that identifier (using the canonical
          identifier if present, otherwise the one issued by <var>issuer</var>) to
          <var>input</var>.
```